### PR TITLE
Add creation of a multi version documentation using sphinx-multiversion

### PR DESCRIPTION
**Pull request by mamueluth**
Part of : https://github.com/ros-controls/control.ros.org/issues/39
Structure is chagned as in [Preparation for multi version](https://github.com/ros-controls/control.ros.org/pull/35) . The creation of the documentation is changed to `sphinx-multiversion`. Currently, three branches are created: Foxy on foxy, galactic on galactic and rolling on master. Since the documentation for e.g. ros2_control is located in separate repositories (subrepositories) the creation of multiversion has been adjusted using scripts located in the `make_help_scripts` folder. The general idea is as follows:
1. For each branch, clone the subrepositories and checkout the branch 
2. Create temporary commits on the branch. Needed since changes in multiversion are only visible after commiting them.
3. Delete temporary commits on branches
4. Create api with doxygen and copy 

For more details, have a look at the Readme.md

### CODE REVIEW

**Review by destogl — CHANGES_REQUESTED** [review]
Just few wording corrections.

Can you please also check that CI is working again? It should execute the same command as when deployment is done, but without push to the `gh-pages` branch:

This path is missing:  `https://github.com/ros-controls/ros2_control/tree/master/transmission_interface/images`

**Review by bmagyar — APPROVED** [review]

File: doc/resources/resources.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ Presentations
   Summary:
     With the release of ros2_control and MoveIt 2, ROS 2 Foxy finally has all the “ingredients” needed to power a robot with similar features as in ROS 1. We present the driver for Universal Robot’s manipulators as a real-world example of how robots can be run using ROS 2. We show how to realize multi-interface support for position and velocity commands in the driver and how to support scaling controllers while respecting factors set on the teach pendant. Finally, we show how this real-world example influences development of ros2_control to support non-joint related inputs and outputs in its real-time control loop.
 
-`Recording <https://vimeo.com/649651707/46a3be27ed>`_
+`Recording <https://vimeo.com/649651707/46a3be27ed>`__
```
[the remark this answers is not in the document]
Comment (mamueluth) [reply doc/resources/resources.rst:37]: Are you sure about this, change? I remember getting a build warning/error because of this

**Comment by mamueluth** [comment]
> Just few wording corrections.
> 
> Can you please also check that CI is working again? It should execute the same command as when deployment is done, but without push to the `gh-pages` branch:
> 
> This path is missing: `https://github.com/ros-controls/ros2_control/tree/master/transmission_interface/images`

The CI should not fail after merge, when all paths are as expected and the correct conf.py is used. I checked this by merging in my own master, there foxy and galactic branches are also present. https://github.com/mamueluth/control.ros.org
The pages are built & deployed without error. https://mamueluth.github.io/control.ros.org/master/index.html

File: doc/resources/resources.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (destogl) [reply doc/resources/resources.rst:37]: Let's keep this as it and and correct later.

**Review by destogl — APPROVED** [review]
Thanks for following up on the issue.

**Pull request merged**